build.gradle

```diff
@@ -0,0 +1,154 @@
+buildscript {
+    repositories {
+        jcenter()
+    }
+    dependencies {
+        classpath "com.jfrog.bintray.gradle:gradle-bintray-plugin:1.7.3"
+    }
+}
+
+plugins {
+    id 'checkstyle'
+    id 'io.spring.dependency-management' version '1.0.3.RELEASE'
+    id 'org.springframework.boot' version '1.5.8.RELEASE'
+}
+
+apply plugin: 'checkstyle'
+apply plugin: 'java-library'
+apply plugin: 'org.springframework.boot'
+apply plugin: 'maven'
+apply plugin: 'maven-publish'
+apply plugin: 'com.jfrog.bintray'
+
+group 'uk.gov.hmcts.reform'
+version '0.0.2'
+
+checkstyle {
+    maxWarnings = 0
+    toolVersion = '8.3'
+    // need to set configDir to rootDir otherwise submodule will use submodule/config/checkstyle
+//    configDir = new File(rootDir, 'config/checkstyle')
+}
+
+sourceCompatibility = 1.8
+
+compileJava {
+    options.compilerArgs << '-parameters' << '-Xlint:deprecation'
+}
+
+compileTestJava {
+    options.compilerArgs << '-Xlint:deprecation'
+}
+
+repositories {
+    maven {
+        url "https://dl.bintray.com/hmcts/hmcts-maven"
+    }
+    jcenter()
+    mavenLocal()
```

Comment:
only needed for local development - perhaps avoid committing?

Reply:
we can take it out once we are happy, however it helps in testing in actual use

Reply:
why would only needed for local development be a reason not to commit it?
it makes it much easier for local, I would recommend keeping it